plugins/statuslines/lualine.nix

```diff
@@ -160,7 +160,10 @@ in {
     };
   in
     mkIf cfg.enable {
-      extraPlugins = [cfg.package];
+      extraPlugins = with pkgs.vimPlugins; [
+        cfg.package
+        nvim-web-devicons
```

Comment:
We should  maybe to this optionally if it's optional in lualine ? For people who want a small config

We could add an option "enableIcons" and conditionally add it.

Reply:
I agree!

Reply:
Done :)